.gitignore

```diff
@@ -1 +1,6 @@
-node_modules
\ No newline at end of file
+node_modules
+.env
+.npm
+npm-debug.log
+build
+.DS_Store
```

Comment:
Missing a new line here. You can modify VS Code to always add a new line at the end of a file.

Reply:
I've added an extension to fix this